.gitignore

```diff
@@ -1,3 +1,2 @@
 /target
 **/*.rs.bk
-Cargo.lock
```

Comment:
I'd rather have a CI action to add the lock to the release. @Luni-4 can you bake the same magic you put on rav1e please? :)